### DISCUSSION TITLE
ImportError: cannot import name 'Transcriber' from 'stt'

### DISCUSSION THREAD

**Discussion by CSLujunyu** [coqui-ai/STT#1824] [Q&A]
I follow the step(Make prediction on multiple audios programmatically), which describes at [https://github.com/mailong25/self-supervised-speech-recognition](https://github.com/mailong25/self-supervised-speech-recognition)

`from stt import Transcriber
transcriber = Transcriber(pretrain_model = 'path/to/pretrain.pt', finetune_model = 'path/to/finetune.pt', 
                          dictionary = 'path/to/dict.ltr.txt',
                          lm_type = 'kenlm',
                          lm_lexicon = 'path/to/lm/lexicon.txt', lm_model = 'path/to/lm/lm.bin',
                          lm_weight = 1.5, word_score = -1, beam_size = 50)
hypos = transcriber.transcribe(['path/to/wavs/0_1.wav','path/to/wavs/0_2.wav'])
print(hypos)`

However, I meet this message(ImportError: cannot import name 'Transcriber' from 'stt').
Could you please provide some solution?

**Comment by reuben** [comment]
> I follow the step(Make prediction on multiple audios programmatically), which describes at https://github.com/mailong25/self-supervised-speech-recognition

That repository is unrelated to this repository. You should start a discussion or issue there.
